app/assets/stylesheets/custom.scss

```
@@ -29,3 +33,7 @@
     border-right: 0;
   }
 }
+
+.form-error {
+  font-size: 0.9rem;
```

```suggestion
  font-size: $small-font-size;
```

So it looks a bit less random :smile:.